### PR TITLE
add Tinkertanker stepper motor

### PR DESCRIPTION
**Pull request by microbit-mark**
cc @yjsoon

### CODE REVIEW

**Comment by abchatra** [comment]
I have filed an issue, this repo requires license. 

Also text of the readme will be displayed as learn more link in MakeCode. Please make sure you are ok with the text. Ideally it should describe what the package is doing and where you can buy the hardware etc.

File: targetconfig.json [hunk 1]
```diff
@@ -150,6 +150,7 @@
             "makecode-extensions/HTS221",
             "assirati/pxt-inventura",
             "Veilkrand/pxt-RobotCar",
+            "Tinkertanker/pxt-stepper-motor"
```
Comment (pelikhan) [inline targetconfig.json]: issing ,

Comment (microbit-mark) [reply targetconfig.json]: fixed

**Comment by abchatra** [comment]
No update here.

**Pull request closed without merging**